sotopia/database/persistent_profile.py

```diff
@@ -36,11 +36,9 @@ class AgentProfile(JsonModel):
     secret: str = Field(default_factory=lambda: "")
     model_id: str = Field(default_factory=lambda: "")
     mbti: str = Field(default_factory=lambda: "")
-    tag: str = Field(
```

Comment:
why remove the tag?

Reply:
Thank you, I'll add the tag back shortly.

Reply:
finished